app/Platform/Models/PlayerAchievement.php

```diff
@@ -16,7 +16,7 @@ class PlayerAchievement extends BasePivot
 
     protected $table = 'player_achievements';
 
-    public const CREATED_AT = 'unlocked_at';
+    public const CREATED_AT = null;
```

Comment:
I believe this causes `SyncPlayerGames` to throw an error, largely because of this in _config/sync.php_:

```php
'player_games' => [
    // ...
    'reference_key' => PlayerAchievement::CREATED_AT,
    // ...
],
```

Perhaps a good `reference_key` value would instead be the `unlocked_at` column?

Reply:
Good catch! Addressed in https://github.com/RetroAchievements/RAWeb/pull/1874/commits/7f20d681045ef05c76819bc54a86f4731aca9498